Prometheus/TextSerializer.cs

```diff
@@ -7,8 +7,15 @@ namespace Prometheus
     /// </remarks>
     internal sealed class TextSerializer : IMetricsSerializer
     {
-        private static readonly byte[] NewLine = new[] { (byte)'\n' };
-        private static readonly byte[] Space = new[] { (byte)' ' };
+        private static readonly byte[] NewLine = { (byte)'\n' };
+        private static readonly byte[] Quote = { (byte)'"' };
+        private static readonly byte[] Equal = { (byte)'=' };
+        private static readonly byte[] Comma = { (byte)',' };
+        private static readonly byte[] Underscore = { (byte)'_' };
+        private static readonly byte[] LBrace = { (byte)'{' };
+        private static readonly byte[] RBraceSp = { (byte)'}', (byte)' ' };
+        private static readonly byte[] Sp = { (byte)' ' };
```

Comment:
I would prefer "Space" as everything else is nicely spelled out. [Just on a personal level, my brain expands Sp to Special]